server/views/pages/induction/workInterests/workInterestTypes.njk

```diff
@@ -67,126 +67,126 @@
             items: [
               {
                 value: "OUTDOOR",
-                checked: form.workInterests.includes("OUTDOOR"),
-                text: "OUTDOOR" | formatInPrisonWorkInterest,
+                checked: form.workInterestTypes.includes("OUTDOOR"),
+                text: "OUTDOOR" | formatJobType,
```

Comment:
This page had the wrong filter (an easy mistake to make given it had `WorkInterest` in its name!).

I might change `formatJobType` to `formatWorkInterestType`. Something for Monday...

Reply:
Tread carefully here ... are you sure its the right filter? (I've not checked), but I do know that `formatJobType` is used for previous work experience (ie. the stuff I'm doing)
I'm not sure if both enums have the same values (`TypeOfWorkExperienceValue` and  `WorkInterestsValue`), but in respect of the filter name, it's definitely not just about "future" work interests.
That might be why the filter was named something fairly loose and generic in this respect - its just a Job Type ? 🤷‍♂️

Reply:
You could have checked in the time it took to write that! 🤣 

Work interest types:
```
OUTDOOR
CLEANING_AND_MAINTENANCE
CONSTRUCTION
DRIVING
BEAUTY
HOSPITALITY
TECHNICAL
MANUFACTURING
OFFICE
RETAIL
SPORTS
EDUCATION_TRAINING
WAREHOUSING
WASTE_MANAGEMENT
OTHER
```

`JobTypeValues` in `formatJobTypeFilter`:

```
OUTDOOR
CLEANING_AND_MAINTENANCE
CONSTRUCTION
DRIVING
BEAUTY
HOSPITALITY
TECHNICAL
MANUFACTURING
OFFICE
RETAIL
SPORTS
WAREHOUSING
EDUCATION_TRAINING
WASTE_MANAGEMENT
OTHER
```

`InPrisonWorkInterestValues` in `formatInPrisonWorkInterestFilter`:
```
CLEANING_AND_HYGIENE
COMPUTERS_OR_DESK_BASED
GARDENING_AND_OUTDOORS
KITCHENS_AND_COOKING
MAINTENANCE
PRISON_LAUNDRY
PRISON_LIBRARY
TEXTILES_AND_SEWING
WELDING_AND_METALWORK
WOODWORK_AND_JOINERY
OTHER
```

You be the judge.

Reply:
Wrt the naming, it makes sense to stick with `formatJobType`, since as you pointed out, it's not just for "future" work 👍